init.lua

```diff
@@ -5,6 +5,8 @@ if minetest.get_translator then S = minetest.get_translator(MOD_NAME) end
 
 treasure_chest = {}
 
+-- local S = minetest.get_translator("treasure_chest")
+
```

Comment:
This should really be removed, not left in, since the translator `S` function is defined above.

Reply:
ahhh a forgotten comment....